cypress/steps/search-column-selector-steps.ts

```diff
@@ -20,6 +20,10 @@ export class SearchColumnSelectorSteps {
     return cy.get('[appCypressData="column-selection-dropdown"]');
   }
 
+  static closeColumnSelectDropdown() {
+    cy.get('.cdk-overlay-backdrop').click(-50, -50, {force: true});
```

Comment:
:D It's OK this way. I just wonder if you need the coordinates and the force in the click here?!

Reply:
Not sure if needed. Comes straight from SO.